lcd/boilerplate/spec/test_spec.rb

```diff
@@ -1,4 +1,60 @@
-describe 'This setup' do
-	it 'is amazing' do
-	end
+require_relative '../lib/lcd'
+require 'pry'
+
+describe Lcd  do
+  describe '#display' do
```

Comment:
Might be better for a business level object like this to have a single behaviour, a verb/action, and to use scenario based language for contexts. Objects with single responsibilities are easier to reason about, maintain, and test as they simpler than an object with doing multiple things with multiple public methods.

This then means you can avoid naming methods in describe statements and avoid the extra level of indentation.